bin/millau/node/src/chain_spec.rs

```diff
@@ -81,8 +81,8 @@ impl Alternative {
 		);
 		match self {
 			Alternative::Development => ChainSpec::from_genesis(
-				"Development",
-				"dev",
+				"Millau",
+				"millau",
```

Comment:
I figured out that i need to change the names as well for Development otherwise UI would not work with development nodes. In case this is wrong i would just revert this change.

Reply:
I agree with @svyatonik, there should be some way to distinguish the development chain spec from the local test net one. Something like;
- `Millau Development` / `millau_dev`

and

- `Millau Local` / `millau_local`